v2/spiffetls/tlsconfig/config.go

```diff
@@ -46,19 +46,21 @@ func HookMTLSClientConfig(config *tls.Config, svid x509svid.Source, bundle x509b
 }
 
 // MTLSWebClientConfig returns a TLS configuration which presents an X509-SVID
-// to the server and verifies the server certificate using system roots.
-func MTLSWebClientConfig(svid x509svid.Source) *tls.Config {
+// to the server and verifies the server certificate using provided roots (or
+// the system roots if nil)
+func MTLSWebClientConfig(svid x509svid.Source, roots *x509.CertPool) *tls.Config {
```

Comment:
We may explicit in the comment what's the role of the x509svid.Source?

Reply:
If we want to do that, let's tackle that in a follow up PR. Doing so will require comment changes on all of the `tlsconfig` methods for consistency.